code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -1071,6 +1071,9 @@
 	set name = "Choose Icon"
 	set waitfor = 0
 
+	if (istype(src, /mob/living/silicon/robot/shell))
```

Comment:
unaddressed

Reply:
Forgot to remove that bit after making the check over in another file. Sorry.